pkg/cache/v2/cluster.go

```diff
@@ -69,11 +72,14 @@ func NewClusterCache(bpfAds *bpfads.BpfAds, hashName *utils.HashName) ClusterCac
 			apiClusterCache[Cluster.Name] = &cluster_v2.Cluster{}
 		}
 	}
+
+	m, _ := maglev.InitMaglevMap(maglevOuter)
```

Comment:
Ignore the error return value here because there is no error?

Reply:
should handle the error